src/inlineImages.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+const fs = require('fs').promises;
+const path = require('path');
+
+let inlineImages = async htmlPath => {
+	const imgTagRegex = /<img (?:.* )?src="([\w.\-\/]+)".*>/;
```

Comment:
```suggestion
	const imgTagRegex = /<img (.* )?src="([\w.\-\/]+)"(.*)>/;
```

Reply:
I want to avoid duplicating the regex string twice to prevent accidentally forgetting to update one of them in the future. What do you think of just using this more genearlized version both at line 10 and 17.

Reply:
Yeah, makes sense